src/array.vala

```diff
@@ -2,70 +2,84 @@ using GLib;
 
 public class Vast.Array : Object
 {
+    /* GType of the scalar elements, immutable */
```

Comment:
You should use multi-line comments with `/**` prefix, otherwise Valadoc won't like it.